app/src/main/java/agency/five/tmdb/ui/components/TabList.kt

```diff
@@ -0,0 +1,107 @@
+package agency.five.tmdb.ui.components
+
+
+import agency.five.tmdb.navigation.Screens
+import agency.five.tmdb.ui.theme.Typography
+import agency.five.tmdb.ui.viewmodel.HomeScreenViewModel
+import android.annotation.SuppressLint
+import android.util.Log
+import androidx.compose.animation.core.FastOutSlowInEasing
+import androidx.compose.animation.core.animateDpAsState
+import androidx.compose.animation.core.tween
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.*
+import androidx.compose.material.TabRowDefaults
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.composed
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.debugInspectorInfo
+import androidx.compose.ui.unit.dp
+import androidx.navigation.NavController
+
+
+@Composable
+fun TabList(
+    tabData: List<String>,
+    moviesLists: List<List<MovieItemViewState>>,
+    navController: NavController,
+    model: HomeScreenViewModel
+) {
+    var tabIndex by remember { mutableStateOf(0) }
+
+    Column(
+        horizontalAlignment = Alignment.Start
+    ) {
+        TabRow(
+            selectedTabIndex = tabIndex,
+            backgroundColor = Color.Transparent,
+            divider = { TabRowDefaults.Divider(thickness = 5.dp, color = Color.Transparent) },
+            indicator = { tabPositions ->
+                TabRowDefaults.Indicator(
+                    modifier = Modifier.customTabIndicatorOffset(tabPositions[tabIndex]),
+                    height = 3.dp,
+                    color = Color(0xFF0B253F)
+                )
+            }
+        ) {
+            tabData.forEachIndexed { index, name ->
+                Tab(
+                    selected = tabIndex == index,
+                    onClick = {
+                        tabIndex = index
+                    },
+                    unselectedContentColor = Color(0xFF828282),
```

Comment:
You can extract colors and dimensions into resource files so that the view doesn't know the color code or specific dimensions.

Add color to the `Color` file or you can add to the` colors.xml` resource file and use with `colorResource (id = R.color.black)`

This is a good practice that needs to be trained :)